payment-methods.adoc

```diff
@@ -0,0 +1,285 @@
+= Payment Methods
+:imagesdir: ./images
+:toc:
+:high-risk:     0.25 BTC
+:mid-risk:      0.50 BTC
+:low-risk:      1.00 BTC
+:very-low-risk: 2.00 BTC
+
+There are several payment methods you can use to buy and sell bitcoin with Bisq. This doc covers the details and limitations of each.
+
+Every offer on Bisq designates a payment method for traders to settle payments. The Bisq software does not actually integrate with any payment methods—all non-bitcoin fund transfers are made outside of the Bisq software.
```

Comment:
Not exactly true since BSQ are also done in-client, although it's not automated and not integrated, might not be worth mentioning.